packages/main.py

```diff
@@ -127,7 +127,7 @@ def handler_midnight(self):
         except Exception:
             log.exception("Fehler im Main-Modul")
 
-    @exit_after(10)
+    @exit_after(60)
```

Comment:
```suggestion
    @exit_after(10)
```
Ich habe die Backup-Erstellung in einen Thread ausgelagert. Damit sollte die Änderung obsolet sein. Du kannst es gerne testen, indem Du ein Rebase auf den aktuellen Master machst.